src/ServiceControl.UnitTests/ApprovalFiles/APIApprovals.PlatformSampleSettings.approved.txt

```diff
@@ -36,6 +36,5 @@
   "MaximumConcurrencyLevel": 10,
   "RetryHistoryDepth": 10,
   "RemoteInstances": [],
-  "DataSpaceRemainingThreshold": 20,
```

Comment:
These platform sample tests are supposed to show when configuration changes in order to make sure the the [config files get updated](https://github.com/Particular/Particular.PlatformSample/tree/master/src/Particular.PlatformSample/configs), but this change shows the test isn't all that effective since the setting is still valid but has now disappeared from this approval file...